packages/block-library/src/gallery/edit.js

```diff
@@ -594,6 +615,43 @@ function GalleryEdit( props ) {
 					) }
 				</PanelBody>
 			</InspectorControls>
+			<BlockControls group="block">
+				<ToolbarDropdownMenu
+					icon={ linkIcon }
+					label={ __( 'Link To' ) }
+				>
+					{ ( { onClose } ) => (
+						<MenuGroup className="blocks-gallery__link-to-control">
+							{ linkOptions.map( ( linkItem ) => {
+								const isOptionSelected =
+									linkTo === linkItem.value;
+								return (
+									<MenuItem
```

Comment:
> @geriux @dcalhoun, If you know why the unit tests are failing, I would appreciate it if you could let me know 🙇‍♂️ This PR moves the link control of the Gallery block from the block sidebar to the block toolbar.

Both `MenuGroup` and `MenuItem` are unavailable for the native mobile editor currently. This results in the crash and unit test failure regarding the `Dropdown` component attempting to render/return an `undefined` value. 

We encountered this situation before and addressed it in https://github.com/WordPress/gutenberg/pull/35191 by bifurcating the code along the web and native platform line. It is not an ideal solution — see https://github.com/WordPress/gutenberg/pull/35191#issuecomment-930245860 — but, unfortunately, the lack of support for `MenuGroup` and `MenuItem` remains the current state. 

If we take a similar approach here, we might retain the previous `SelectControl` for native platforms. I opened https://github.com/WordPress/gutenberg/pull/63953 implementing this option, if we'd like to use this approach to move the proposed web changes forward quickly. 

Unfortunately, I do not believe the native mobile team will prioritize `MenuGroup` or `MenuItem` components in the near future.

Reply:
@dcalhoun appreciate your response. Thank you for the details on the issue and PR too.

Reply:
@t-hamano is it worth doing this way?